sky/utils/common_utils.py

```diff
@@ -626,3 +626,20 @@ def fill_template(template_name: str, variables: Dict,
     content = j2_template.render(**variables)
     with open(output_path, 'w', encoding='utf-8') as fout:
         fout.write(content)
+
+
+def deprecated_function(func: Callable, name: str, deprecated_name: str,
```

Comment:
is there other deprecated function in the system? if so, change to this decorator as well?

Reply:
No, I don't think there is any other deprecated functions in the system.